test/functional/test_framework/mempool_util.py

```diff
@@ -57,16 +57,17 @@ def fill_mempool(test_framework, node, *, tx_sync_fun=None):
     # Generate UTXOs to flood the mempool
     # 1 to create a tx initially that will be evicted from the mempool later
     # 75 transactions each with a fee rate higher than the previous one
+    # 1 to create a final middle-sized tx to evict a filler tx and create room (if needed)
     ephemeral_miniwallet = MiniWallet(node, tag_name="fill_mempool_ephemeral_wallet")
-    test_framework.generate(ephemeral_miniwallet, 1 + num_of_batches * tx_batch_size)
+    test_framework.generate(ephemeral_miniwallet, 2 + num_of_batches * tx_batch_size)
```

Comment:
same, can we extract the `2` to a named constant?

Reply:
I think the comment immediately above explains it sufficiently, and the "1" was not a named constant either (so this pull is not making it worse). But if you feel strongly, I can do something. (As someone once said, the hardest part of programming is coming up with good names!)